content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,498 @@
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+```sh
+curl -X POST \
+         "https://app.datadoghq.com/api/v2/authn_mappings/{UUID}" \
+         -H "Content-Type: application/json" \
+         -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+         -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>" \
+         -d '{
+             "data": {
+                 "type": "authn_mappings",
+                 "id": "123e4567-e89b-12d3-a456-426655440000",
+                 "attributes": {
+                      "role_uuid": "123e4567-e89b-12d3-a456-426655445555",
+                      "attribute_key": "string",
+                      "attribute_value": "string"
+                }
+             }
+         }'
+```
+{{< /code-block >}}
+
+Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeholder with the corresponding [API and application keys for your organization][1].
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "Response" %}}
+
+{{< code-block lang="json" filename="response.json" disable_copy="true" >}}
+{
+  "data": {
+    "attributes": {
+      "created_at": "timestamp",
+      "created_by": "string",
+      "role_uuid": "00000000-0000-0000-0000-000000000000",
+      "saml_assertion_attribute_id": 0
+    },
+    "type": "authn_mappings",
+    "id": "string",
+    "relationships": {
+      "saml_assertion_attributes": {
+        "data": {
+          "id": 0,
+          "type": "saml_assertion_attributes"
+        }
+      },
+      "roles": {
+        "data": {
+          "id": "123e4567-e89b-12d3-a456-426655440000",
+          "type": "roles"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "data": {
+        "id": "string",
+        "type": "roles",
+        "attributes": {
+          "created_at": "string",
+          "modified_at": "string",
+          "uuid": "123e4567-e89b-12d3-a456-426655440000",
+          "name": "string"
+        },
+        "relationships": {
+          "data": [
+            {
+              "id": "string",
+              "type": "permissions"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "data": {
+        "id": 0,
+        "type": "saml_assertion_attributes",
+        "attributes": {
+          "id": 6,
+          "attribute_key": "string",
+          "attribute_value": "string"
+        }
+      }
+    }
+  ]
+}
+{{< /code-block >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Get all AuthN Mappings
+
+Returns a list of AuthN Mappings
+
+| Method | Endpoint path            | Required payload |
+|--------|-------------------------|------------------|
+| `GET`  | `/v2/authn_mappings` | optional queries       |
+
+##### ARGUMENTS
+
+* **`sort`** [*optional*, *default*=**yes**]:
+  Sort attribute and direction—`yes` sorts in descending order, `no` sorts in ascending order.
```

Comment:
The sort param looks something like this 
api/v2/authn_mappings?sort=created_at

Where by default, attributes are sorted ascending. To do descending, you just pass a negative sign. So
api/v2/authn_mappings?sort=-created_at
would sort in the descending direction